sources/packages/api/src/route-controllers/user/models/institution-user.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsNotEmpty } from "class-validator";
+
+export class InstitutionUserDto {
+  @IsNotEmpty()
```

Comment:
The class validators should be used to validate input DTOs, they have effect on response DTOs.
Please remove the class validators.

Reply:
Class validator used only for persist DTO